selection_tools/print_number_of_events.C

```diff
@@ -0,0 +1,196 @@
+
+double B(vector<double> n_vec){
+  double n_tot = 0;
+  for (double n : n_vec){
+    n_tot += n;
+  }
+  return n_tot;
+}
+
+double B_sigma(vector<double> sigma_vec){
+  double sigma2 = 0;
+  for (double s : sigma_vec){
+    sigma2 += s*s;
+  }
+  return sqrt(sigma2);
+}
+
+vector<double> figure_of_merit(vector<double> sig_events, double bkg_n, double bkg_sigma){
```

Comment:
A small thing, but improves readability: this function returns a figure of merit, so it would make more sense to call it `get_figure_of_merit`.

Then there's a different problem - if it returns a figure of merit, why is it a vector? Maybe it actually returns something else, like `get_figure_of_merit_per_signal`?

Finally, a "figure of merit" doesn't actually tell us what it calculates. A figure of merit could be anything, so it's better to be specific and say `get_significance_per_signal`.